python/test/test_dataset.py

```diff
@@ -465,6 +465,26 @@ def test_Workspace2D_example(self):
                 spec[Data.Value, "sample1"] = np.zeros(1000)
                 spec[Data.Variance, "sample1"] = np.zeros(1000)
 
+    def test_zip(self):
+        d = Dataset()
+        d[Coord.SpectrumNumber] = ([Dim.Position], np.arange(1, 6))
+        d[Data.EventTofs, ""] = ([Dim.Position], (5,))
```

Comment:
Dim.Position for both SpectrumNumber and TOF? Seems to make sense for SpectrumNumber based on underlying type (Vector3d).

Reply:
Not sure what your question is. I chose `Dim.Position` since essentially that is what the spectra in Mantid typically are --- data for different positions. I may just look a bit confusing here because the corresponding axis, `Coord.Position` is not added to the dataset.

Reply:
From a usage point of view, i.e slicing this makes sense, though defining the dataset like this doesn't feel intuitive to me yet. I think that's why I originally had trouble understanding line 471. I'm not suggesting changing this.

Reply:
We can add initialization from lists/tuples of variables if that is more intuitive? Or did you have something else in mind?

Reply:
It does make sense when you understand how the event data is being stored, i.e the underlying type for the tag. I think it takes some getting used to. Would be interested in what you had in mind along the tuple route?

Reply:
Something like `xarray`'s way: http://xarray.pydata.org/en/stable/data-structures.html#creating-a-dataset